src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx

```diff
@@ -264,6 +264,7 @@ function TransactionWorkflow({
       switch (contractType) {
         case CONTRACT_TYPES.ANT: {
           switch (interactionType) {
+            case ANT_INTERACTION_TYPES.SET_TICKER:
```

Comment:
interesting we're using a switch here but likely man of these will fall through to do the same thing. let's consider just doing an if check. switches within switches is tough to maintain and comprehend